great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -58,11 +58,11 @@ def __init__(
         metric_name: str,
         metric_domain_kwargs: Optional[Union[str, dict]] = None,
         metric_value_kwargs: Optional[Union[str, dict]] = None,
-        sampling_method: str = "bootstrap",
         enforce_numeric_metric: Union[str, bool] = True,
         replace_nan_with_zero: Union[str, bool] = True,
         reduce_scalar_metric: Union[str, bool] = True,
         false_positive_rate: Union[str, float] = 5.0e-2,
+        sampling_method: str = "bootstrap",
```

Comment:
What are our different options here? Does this highlight a need for enums?

Reply:
@cdkini Right now only `bootstrap` and `oneshot` -- the later for testing purposes only (never to be used in production).